### DISCUSSION TITLE
타 클래스에서 GameObject가 Destroy되었는지 확인하는 방법이 궁금합니다.

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#466] [5. Q&A]
아래 예제 코드처럼 작성해서 사용하고 있는데, GameObject가 파괴된 이후에도 조건문을 통과하고 있습니다.
혹시 GameObject가 파괴되었는지 코드에서 확인할 수 있는 방법이 있을까요?

```ts
// gameObjectA 제거
GameObject.Destroy(this.gameObjectA);

// gameObjectA 가 활성 오브젝트인지 확인
if(this.gameObjectA) {
  console.warn("이 조건문으로 게임오브젝트가 파괴되었는지 확인할 수 없습니다.");
}

```

**Comment by minjujuu** [comment]
Equals(null) 로 한 번 사용해보세요!
```
        if(this.gameObjectA.Equals(null) == false) {
            console.log(`this.gameObjectA가 존재`);
        }
```

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요. 
@minjujuu 님이 말씀해 주신 것처럼  if(this.gameObjectA.Equals(null) == false) 을 사용하면 파괴 상태가 감지되는 것을 구현할 수 있습니다.
다만 파괴된 이후에도 if(this.gameObjectA) 사용시 true로 감지되는 부분은
내부에서 살펴보도록 하겠습니다.
